src/rSharp.cpp

```diff
@@ -36,7 +36,10 @@ void freeObject(RSharpGenericValue* instance);
 /////////////////////////////////////////
 // Initialization and disposal of the CLR
 /////////////////////////////////////////
-void rSharp_create_domain() {
+void rSharp_create_domain(char ** libPath)
+{
+	char_t* wideStringPath = MergeLibPath(libPath, "/RSharp.runtimeconfig.json");
+	dotnetlib_path = MergeLibPath(libPath, "/ClrFacade.dll");

```

Comment:
both constant strings on this line and the line above could be STR("the string in here") because then you don't have to allocate memory for the widening conversion. They will be compiled as wchar_t

Reply:
Also, move this below the blocking test for nullptr. We don't need to do any string concatenation if the assembly pointer is already loaded.